src/database/simple.jl

```diff
@@ -394,6 +394,34 @@ const renames = [
     @add_rename iteratoreltype IteratorEltype   v"0.7.0-DEV.3309"
     @add_rename nb_available   bytesavailable   v"0.7.0-DEV.3477"
     @add_rename broadcast_indices broadcast_axes  v"0.7.0-DEV.4936"
+    @add_rename Base.Test      Test             v"0.7.0-alpha" 
+    @add_rename LinAlg         LinearAlgebra    v"0.7.0-alpha"
+    @add_rename bits           bitstring        v"0.7.0-alpha"
+    @add_rename search         findfirst        v"0.7.0-alpha"
+    @add_rename rsearch        findlast         v"0.7.0-alpha"
+    @add_rename Random.dSFMT   Random.DSFMT     v"0.7.0-alpha"
+    @add_rename gc             GC.gc            v"0.7.0-alpha"
+    @add_rename gc_enable      GC.enable        v"0.7.0-alpha"
+    @add_rename Base.@gc_preserve GC.@preserve  v"0.7.0-alpha"
+    @add_rename LinSpace      LinRange          v"0.7.0-alpha"
+    @add_rename endof         lastindex         v"0.7.0-alpha"
+    @add_rename DateTime()    DateTime(1)       v"0.7.0-alpha"
```

Comment:
Same here, this is not an identifier. You can write a custom AST matcher instead.